uberpoet/multisuite.py

```diff
@@ -183,33 +190,29 @@ def build_app_type(self, gen_type, wmo_enabled):
         # Build App
         total_time = 0
         if self.run_xcodebuild:
-            logging.info('Generate Xcode workspace & clean')
+            logging.info('Generate workspace & clean')
 
             derived_data_path = join(tempfile.gettempdir(), 'ub_mockapp_derived_data')
             shutil.rmtree(derived_data_path, ignore_errors=True)
             makedir(derived_data_path)
 
-            if self.project_generator_type == "buck":
```

Comment:
the `-d` option was not working for me. Furthermore this is another case in which we generated an xcodeproj with buck but when we built it was never using buck.

I removed this entirely. Let me know if that is OK.

Reply:
Yeah building with xcodebuild was the original intention, since we only had one project generator type, maybe add xcodebuild as an option to multisuite later?  I'm fine with leaving it to build with buck / bazel directly for now, people can use cocoapods to test 'direct' xcodebuild.